src/lib/components/works_for_you/__tests__/__snapshots__/notifications-tests.tsx.snap

```diff
@@ -6,11 +6,24 @@ exports[`test does not show artist avatar if no avatar image exists 1`] = `
     }
   }>
   <View
+    accessibilityComponentType={undefined}
```

Comment:
this all comes from the new artist `TouchableWithoutFeedback` on each notification.